rpc/common/service.proto

```diff
@@ -21,7 +21,7 @@ message PackageInfo {
 message Application {
   string           type      = 1;
   string           file_path = 2;
-  repeated Library libraries = 3;
+  repeated Package libraries = 3;
```

Comment:
Could you test if an old client can communicate with a new server and vice versa? The id 3 is overwritten and I'm worried about compatibility.

Reply:
I've run `trivy` in server mode from new sources, and tried to scan an image from `trivy` client v0.22.0. 
it works.

I thought that these changes aren't a problem, because `Package` struct contains all fields from `Library`. but maybe i missed something

Reply:
Did the image contain language-specific packages?

Reply:
I test `logstash:7.16.2`. this image contains `log4j`

Reply:
Did it work in your environment? I got the following errors.

```
2022-02-03T11:12:20.604+0200    WARN    Error while getting vulnerability details: failed to get the vulnerability: failed to marshal JSON: unexpected end of JSON input

2022-02-03T11:12:20.605+0200    WARN    Error while getting vulnerability details: failed to get the vulnerability: failed to marshal JSON: unexpected end of JSON input

2022-02-03T11:12:20.982+0200    WARN    Error while getting vulnerability details: failed to get the vulnerability: failed to marshal JSON: unexpected end of JSON input

2022-02-03T11:12:20.983+0200    WARN    Error while getting vulnerability details: failed to get the vulnerability: failed to marshal JSON: unexpected end of JSON input

2022-02-03T11:12:20.983+0200    WARN    Error while getting vulnerability details: failed to get the vulnerability: failed to marshal JSON: unexpected end of JSON input

2022-02-03T11:12:21.151+0200    WARN    Error while getting vulnerability details: failed to get the vulnerability: failed to marshal JSON: unexpected end of JSON input

2022-02-03T11:12:21.662+0200    WARN    Error while getting vulnerability details: failed to get the vulnerability: failed to marshal JSON: unexpected end of JSON input

2022-02-03T11:12:21.815+0200    WARN    Error while getting vulnerability details: failed to get the vulnerability: failed to marshal JSON: unexpected end of JSON input

2022-02-03T11:12:21.828+0200    WARN    unknown severity:
2022-02-03T11:12:21.828+0200    WARN    unknown severity:
2022-02-03T11:12:21.829+0200    WARN    unknown severity:
2022-02-03T11:12:21.829+0200    WARN    unknown severity:
2022-02-03T11:12:21.829+0200    WARN    unknown severity:
2022-02-03T11:12:21.829+0200    WARN    unknown severity:
2022-02-03T11:12:21.833+0200    WARN    unknown severity:
```

Reply:
@knqyf263 it's a really strange. I can't reproduce this errors.
my steps are next:
```sh
# Build my custom trivy
$ go build -o tr cmd/trivy/main.go

# Run trivy in server mode
$ ./tr -d server --listen localhost:8080
```
and then testing:
```sh
$ ./tr -d client --remote http://localhost:8080 logstash:7.16.2
```

Reply:
I'm able to reproduce the same `Error while getting vulnerability details: failed to get the vulnerability: failed to marshal JSON: unexpected end of JSON input` error with `ruby:2.6-slim` and `ruby:2.7-slim`:
```
trivy image -f json -o report.json ruby:2.6-slim
```

Reply:
@dinvlad oh, thanks for report! will try too

Reply:
@dinvlad me too. I could reproduce an error for `ruby:2.6-slim`.
It seems that there is another problem...
could you create a new issue for this bug? thanks a lot

Reply:
> ```
> 2022-02-03T11:12:21.815+0200    WARN    Error while getting vulnerability details: failed to get the vulnerability: failed to marshal JSON: unexpected end of JSON input
> 
> 2022-02-03T11:12:21.828+0200    WARN    unknown severity:
> ```

It seems that this issue isn't related to this PR.
the current `trivy` v0.23.0 contains the same problem.

Reply:
Sounds good - opened https://github.com/aquasecurity/trivy/issues/1691

Reply:
> 2022-02-03T11:12:21.815+0200    WARN    Error while getting vulnerability details: failed to get the vulnerability: failed to marshal JSON: unexpected end of JSON input
> 2022-02-03T11:12:21.828+0200    WARN    unknown severity:

I can see such messages, but It doesn't affect on the result. right?